app/models/menu/menu.rb

```diff
@@ -502,59 +502,66 @@ def hmis_admin_menu
     menu = Menu::Item.new(
       user: user,
       title: Translation.translate('HMIS Admin'),
+      icon: 'icon-cog',
       match_pattern_terminator: '.*',
     )
     # If we don't have the ENV for HMIS, the path isn't available
     # Checking enforcement outside keeps it from throwing an error
     if HmisEnforcement.hmis_enabled?
-      menu.add_child(
+      access_menu = Menu::Item.new(
+        user: user,
+        title: 'Access',
+        id: 'hmis-access',
+      )
+      access_menu.add_child(
         Menu::Item.new(
           user: user,
           visible: ->(_user) { context.hmis_admin_visible? },
           path: hmis_admin_access_overviews_path,
           title: Translation.translate('Overview'),
         ),
       )
-      menu.add_child(
+      access_menu.add_child(
```

Comment:
I can back this out, it should match the warehouse. This is an order that makes sense to me, but that might just be me!